README.md

```diff
@@ -60,7 +60,7 @@ The `Kubernetes.secrets` plugin enables resolution of credential or parameter va
 
     [[secrets]]
     name = "kubernetes-secrets"
-    plugin = "kubernetes.secret"
+    plugin = "kubernetes.secrets"
     ```
 
 * If the plugin is being used outside of a Kubernetes cluster then add the following lines to specify the namespace to be used to store data:
```

Comment:
Can you add something to the README that explains how this plugin works? It's always been a bit confusing:

* What namespace is used?
* What should the secret name be in k8s?
* What should the secret data keys be? How does that match up with a secret referenced in a credential or parameter set?